aspnetcore/zone-pivot-groups.yml

```
@@ -0,0 +1,12 @@
+# YamlMime:ZonePivotGroups
+groups:
+- id: operating-systems
+  title: Tooling
+  prompt: Choose an operating system or use a command shell
```

weren't you going to remove CLI?

```suggestion
  prompt: Choose an operating system
```